content/en/real_user_monitoring/guide/proxy-mobile-rum-data.md

```diff
@@ -48,7 +48,8 @@ For more information, see the [OkHttpClient Proxy and Authenticator][2] document
 {{% tab "iOS" %}}
 
 When initializing the iOS SDK, specify the following proxy configuration:
-
+{{< tabs >}}
+{{% tab "Swift" %}}
```

Comment:
```suggestion
```

Tabs within tabs aren't supported, but we could add either a Header for Swift and Objective C, or place these within code blocks like this:
```
{{< code-block lang="swift" filename="Swift Configuration" disable_copy="true" collapsible="true" >}}

import DatadogCore

Datadog.initialize(
  with: Datadog.Configuration(
    clientToken: "<client token>",
    env: "<environment>",
    proxyConfiguration: [
        kCFNetworkProxiesHTTPEnable: true,
        kCFNetworkProxiesHTTPPort: <123>,
        kCFNetworkProxiesHTTPProxy: "<www.example.com>",
        kCFProxyUsernameKey: "<proxy user>",
        kCFProxyPasswordKey: "<proxy password>"
    ]
  ),
  trackingConsent: trackingConsent
)
{{< /code-block >}}
```
to look like 
![image](https://github.com/DataDog/documentation/assets/41168354/d17e9d67-49f3-4e76-943d-a5b747c0ce96)